<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QWidget" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxTrafficLightControlGB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>301</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="title">
    <string>交通灯控制</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>路口选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>模式选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>操作选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="roadSelectCB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>路口1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>路口2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>路口3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>路口4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>路口5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="modeSelectCB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="lightTimeSelectCB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>红灯1 s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红灯5 s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红灯10 s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红灯30 s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>红灯60 s</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="applyBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDisplayGB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="title">
    <string>显示控制</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>221</width>
      <height>79</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxLocationInfo">
       <property name="text">
        <string>定位信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxDataInfo">
       <property name="text">
        <string>流量信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxRoadInfo">
       <property name="text">
        <string>路口信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCarControlGB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
    </font>
   </property>
   <property name="title">
    <string>车辆控制</string>
   </property>
   <widget class="QLineEdit" name="lineEditShowInstructions">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBackSpace">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退格</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGoAhead">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTurnLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>左转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTurnRight">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>右转</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonOneCar">
       <property name="text">
        <string>1号车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonTwoCar">
       <property name="text">
        <string>2号车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonThreeCar">
       <property name="text">
        <string>3号车</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>223</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButtonFourCar">
       <property name="text">
        <string>4号车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonFiveCar">
       <property name="text">
        <string>5号车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonSixCar">
       <property name="text">
        <string>6号车</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>lineEditShowInstructions</zorder>
   <zorder>pushButtonClear</zorder>
   <zorder>pushButtonBackSpace</zorder>
   <zorder>pushButtonSend</zorder>
   <zorder>pushButtonGoAhead</zorder>
   <zorder>pushButtonTurnLeft</zorder>
   <zorder>pushButtonStop</zorder>
   <zorder>pushButtonTurnRight</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
